README.md

```diff
@@ -1 +1,4 @@
 # tools-telemetry-service
+
+A service to receive telemetry events and pass them on to a kinesis stream.
```

Comment:
Is there anything more specific that could be added here, or are we too early in development of this service to expand?

Reply:
We could be more specific now! I'll add some more text. We're about to add code for the lambda and some interfaces, which will come with more information about behaviour/contracts etc.